webapp/src/containers/Rhs/index.tsx

```diff
@@ -256,14 +256,14 @@ const Rhs = (): JSX.Element => {
                 </div>
             )}
             <Dialog
-                description={dialogContent}
```

Comment:
why this change?

Reply:
Updated Dialog component now has content in children. This prop is not supported